cypress/e2e/bulk-edit/csv/bulk-edit-csv-drag-and-drop-is-enabled-after-file-is-uploaded.cy.js

```diff
@@ -0,0 +1,63 @@
+import getRandomPostfix from '../../../support/utils/stringTools';
+import { DevTeams, Permissions, TestTypes } from '../../../support/dictionary';
+import TopMenu from '../../../support/fragments/topMenu';
+import Users from '../../../support/fragments/users/users';
+import BulkEditSearchPane from '../../../support/fragments/bulk-edit/bulk-edit-search-pane';
+import FileManager from '../../../support/utils/fileManager';
+
+describe('bulk-edit', () => {
+  const testData = {};
+  const userUUIDsFileName = `userUUIDs_${getRandomPostfix()}.csv`;
+  const invalidUserUUID = getRandomPostfix();
```

Comment:
Why invalidUserUUID? In the test you should use  .csv file with valid User UUIDs.

Reply:
got it. Though deleted those completely, since according to the second comment now using file directly from the fixtures